modbus4mqtt/modbus_interface.py

```diff
@@ -226,11 +226,18 @@ def _scan_value_range(self, table, start, count):
             result = self._mb.read_input_registers(start, count, unit=self._unit)
         elif table == 'holding':
             result = self._mb.read_holding_registers(start, count, unit=self._unit)
+        elif table == 'do':
+            result = self._mb.read_do(start, count, unit=self._unit)
```

Comment:
`read_do`  do not exists, you would like to call `read_coils` instead.
[PyModBus docs](https://pymodbus.readthedocs.io/en/v3.5.4/source/client.html)

![image](https://github.com/tjhowse/modbus4mqtt/assets/100710244/a5347882-89f4-447e-9199-37e155f3e242)